src/MoBi.Presentation/Presenter/EditParameterListPresenter.cs

```diff
@@ -91,6 +96,14 @@ public virtual IReadOnlyList<IParameter> SelectedParameters
          }
       }
 
+      public void EnableSimulationTracking(TrackableSimulation simulation) => _trackableSimulation = simulation;
+
+      public bool HasBuildingBlocks(IReadOnlyList<ParameterDTO> parameterDTOs) => parameterDTOs.Any(parameterDTO => !string.IsNullOrEmpty(parameterDTO.BuildingBlockName));
```

Comment:
why are we passing the list of parmaeterDTOs here? isn't it _parameterDTOs ? (private member)?